jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/select/CustomTriggerTextSelectTests.java

```diff
@@ -0,0 +1,50 @@
+package io.github.epam.angular.tests.elements.complex.select;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+
+import static io.github.com.pages.sections.SelectSection.customTriggerTextSelect;
+
+public class CustomTriggerTextSelectTests extends TestsSelectBase {
+    @BeforeMethod()
+    public void before() {
+        customTriggerTextSelect.show();
+    }
+
+    @Test
+    public void checkLabelValue() {
+        customTriggerTextSelect.label().has().value("Toppings");
+    }
+
+    @Test
+    public void checkOptionCanBeSelectedByName() {
+        customTriggerTextSelect.multipleSelect(SAUSAGE);
+        customTriggerTextSelect.is().selected(SAUSAGE);
+        customTriggerTextSelect.multipleSelect(SAUSAGE);
+    }
+
+    @Test
+    public void checkAllOptionsCanBeSelectedById() {
+        customTriggerTextSelect.multipleSelect(1, 2, 3, 4, 5, 6);
+        customTriggerTextSelect.verify().selected(EXTRA_CHEESE + " (+5 others)");
+        customTriggerTextSelect.multipleSelect(1, 2, 3, 4, 5, 6);
+    }
+

```

Comment:
Could you please explain. Why verify() method is using instead of assert. And why  multipleSelect(1, 2, 3, 4, 5, 6) method is using twice.

Reply:
I used the verify() method only as an alternative, since it is an alias of the is() method.
The second call of the multipleSelect(1, 2, 3, 4, 5, 6) method is performed as a cleanup.